src/settings/constants/sets.js

```diff
@@ -32,4 +32,5 @@ export const SET_FIELDS = {
 export const SET_FIELDS_INITIAL_VALUES = {
   [SET_FIELDS.NAME]: '',
   [SET_FIELDS.DESCRIPTION]: '',
+  [SET_FIELDS.SET_SPEC]: 'DEFAULT_SET_SPEC', // TODO:: Temporary solution for integration with BE. Change from 'DEFAULT_SET_SPEC' to '' after SET_SPEC implantation.
```

Comment:
Is there a back-end JIRA issue number you can reference here?

Reply:
On the BE side, this is already a required field and cannot be empty, but on the UI side this field will be implemented in future sprints.

Reply:
@ryandberger Could you please re-review?